core/domain/html_validation_service_test.py

```diff
@@ -921,15 +918,16 @@ def test_validate_customization_args(self):
             'Missing attributes: [u\'alt-with-value\'], Extra attributes: []':
             [(
                 '<oppia-noninteractive-image caption-with-value="&amp;quot;abc'
-                '&amp;quot;" filepath-with-value="&amp;quot;random.png&amp;'
-                'quot;"></oppia-noninteractive-image>'
+                '&amp;quot;" image_id-with-value="&amp;quot;image_id_1&amp;quot;"'
+                '></oppia-noninteractive-image>'
             )],
             'Video id length is not 11': [(
                 '<oppia-noninteractive-video autoplay-with-value="false" '
                 'end-with-value="0" start-with-value="0" video_id-with-value="'
                 '&amp;quot;lorem&amp;quot;"></oppia-noninteractive-video>'
             )]}
 
+        # self.assertEqual(set(actual_output.keys()), "keshav")
```

Comment:
Please remove this code. In general, go through the files tab after committing some changes and make sure your PR is ready for a review by others.)

Reply:
Is this resolved?